kod/object/active/holder/room/rentroom.kod

```diff
@@ -1230,7 +1234,7 @@ messages:
       % Everything except for 6th category is fine.
       if NOT Send(Send(SYS,@GetNodeAttack),@GetAttacks)
       {
-         if category <> 6
+         if category <> 11
```

Comment:
There are only 6 categories of objects, category 6 needs to check for all heartstones.